TestBenches/InputRouter_test.cpp

```diff
@@ -612,5 +671,11 @@ int main(int argc, char * argv[])
   }
   cLinkDataStream.close();
 
+  // remove temp LUT 
+  if( remove( "emData/LUTs/IR_Nmems.tab" ) == 0 )
+  {
+    std::cout << "Temperorary LUT created by IR test bench deleted.\n";
+  }
+
```

Comment:
@sseifeln I guess you should remove "emData/IR_Nmems.tab", although again, I'm not sure what the purpose of this file is since the test bench both creates and removes it.

Reply:
Also removed (again). Sorry!